### PR TITLE
NWPS-1465 - Quote block component

### PR DESCRIPTION
**Pull request by AlexKirstenZA**
@sankar-shunmuga @smanivasag @FranciscoRuizHEE 

This PR is for the new Quote component.

Thanks :slightly_smiling_face:

### CODE REVIEW

File: app/views/blocks/content/main-quote.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+<h3>Example</h3>
+{{ example({
+    view: 'block',
+    url: basePath + '/blocks/content/examples/' + page_url,
+    height: '620'
+}) }}
+
+<hr />
+
+<h3>Macro</h3>
```
Comment (smanivasag) [inline app/views/blocks/content/main-quote.html:57]: @AlexKirstenZA - could we fill this in with macro?

File: app/views/blocks/content/main-quote.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  html: '
+
+  '
+}) }}
+
+<hr />
+
+<h3>HTML</h3>
+{{ highlight({
+    html : '
```
Comment (smanivasag) [inline app/views/blocks/content/main-quote.html:68]: @AlexKirstenZA - could we fill this in with HTML snippet?

**Comment by smanivasag** [comment]
@AlexKirstenZA - Could you include a `Quote` block in an example template (e.g. SCP, etc) for reference? I think the block requires additional padding/margin (like other components). Thanks!

File: app/views/blocks/content/main-quote.html (hunk 2 above)
Comment (sankar-shunmuga) [reply app/views/blocks/content/main-quote.html:71]: Resolved

**Review by sankar-shunmuga — APPROVED** [review]
thanks @AlexKirstenZA , approved

**Pull request merged**